src/modules/Packages.rb

```diff
@@ -2702,6 +2700,16 @@ def format_missing_resolvables(type, list)
         _("These items (%{type}) need to be selected to install: %{list}") % {type: type, list: list}
       end
     end
+
+    def fs_mountpoint(filesystem)
+      filesystem.mountpoints[0] || ""
```

Comment:
The concept of "main mount point" is unclear.

With btrfs subvolumes enter the game things will change. How I cannot say yet - maybe each subvolume gets an object in the devicegraph and has no, one or several mount points.